debug.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+CFLAGS='-Wall -std=c11 -Wextra -Werror -pedantic -Og -ggdb'
```

Comment:
I'd move `debug.sh` to a subdirectory such as `dev`, `scripts`.
Item 3 above would have avoided the duplication of `-std` and `-pedantic` here.

Reply:
I'll head this. Clarity can't hurt.